config/modules/acme/default.nix

```diff
@@ -1,19 +1,55 @@
-{ config, ... }:
+{ config, lib, pkgs, ... }:
+
+with lib;
+
 {
-  users.groups.acme = { };
+  options = with types; {
+    security.acme = {
+      nameserver = mkOption {
+        type = str;
+        description = ''
+          Nameserver to push RFC2136 updates to.
+        '';
+      };
+      tsig.key = mkOption {
+        type = str;
+        description = ''
+          Name of the secret key as defined in DNS server configuration.
+        '';
+      };
+      tsig.secret = mkOption {
+        type = str;
```

Comment:
Can we use a path to a secret instead? We shouldn't advocate putting secrets in the store.

Reply:
For ACME, we can. But not for knot. 

I think (currently) there is no way to prevent the secret from ending up in the store. To enable sharing the secret between ACME client and knot resolver, I would prefer the make it directly accessible here.

Reply:
Not supports include statements like so:

```
include: /var/lib/knot/tsig-keys/*.key
```